make/conf/jib-profiles.js

```diff
@@ -419,9 +419,11 @@ var getJibProfilesProfiles = function (input, common, data) {
         "linux-x64": {
             target_os: "linux",
             target_cpu: "x64",
-            dependencies: ["devkit", "gtest", "graphviz", "pandoc"],
-            configure_args: concat(common.configure_args_64bit,
+            dependencies: ["devkit", "gtest", "build_devkit", "graphviz", "pandoc"],
+            configure_args: concat(
+                (input.build_cpu == "x64" ? common.configure_args_64bit : "--openjdk-target=x86_64-linux-gnu"),
                 "--with-zlib=system", "--disable-dtrace",
+                (input.build_cpu != "x64" ? "--openjdk-target=x86_64-linux-gnu" : null),
```

Comment:
Doesn't this add `--openjdk-target=x86_64-linux-gnu` twice when build_cpu is not "x64"?

Reply:
You are correct, I never finished that edit. Uploaded fix.